modules/modlauncher/src/main/java/org/gotti/wurmunlimited/modsupport/CreatureTemplateBuilder.java

```diff
@@ -371,8 +371,11 @@ public CreatureTemplate build() {
 				temp.setOnFire(onFire);
 				temp.setFireRadius(fireRadius);
 			}
+			
+			if (combatMoves != null) {
```

Comment:
Thats fine I won't try to contribute anymore. It works, I can read it, if you can't read it you probably should not be coding.  I will just make my own branch that actually gets changes instead of months without hearing a thing. Wish I never bothered putting this.